Kernel/Time/TimeManagement.h

```diff
@@ -102,9 +100,8 @@ class TimeManagement {
     Atomic<u32> m_update1 { 0 };
     u32 m_ticks_this_second { 0 };
     u64 m_seconds_since_boot { 0 };
-    // FIXME: Should use AK::Time internally
-    timespec m_epoch_time { 0, 0 };
-    timespec m_remaining_epoch_time_adjustment { 0, 0 };
+    UnixDateTime m_epoch_time {};
+    Duration m_remaining_epoch_time_adjustment {};
```

Comment:
yup!

Reply:
I'm not sure what you mean; in this case the types are 100% correct now.